curvefs/src/metaserver/metastore.h

```diff
@@ -107,6 +123,9 @@ class MetaStore {
     virtual bool GetPartitionInfoList(
                             std::list<PartitionInfo> *partitionInfoList) = 0;
 
+    virtual bool GetPartitionSnap(
```

Comment:
GetAllPartitions ?

Reply:
`PartitionSnap` indicates the current partition list.